internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/pubsub/adapter/kafka/producer/ApacheKafkaProducerAdapterITest.java

```diff
@@ -313,7 +313,7 @@ public void testProducerCloseCanUnblockSendMessageCallerThread(boolean doFlush)
       countDownLatch.await();
       // Still wait for some time to make sure blocking sendMessage is inside kafka before closing it.
       Utils.sleep(50);
-      producerAdapter.close(5000, doFlush);
```

Comment:
should we rename `doFlush` to `unboundedTimeout`?

Reply:
Calling `close` with timeout zero is the already-supported way to do the equivalent of `!doFlush`. But I am not sure what your suggestion is? Can you elaborate?

Reply:
My suggestion is mainly about renaming the variable to something like `isUnboundedTimeout` in the test method, since we don't have an **explicit** flush anymore in the close method.